ci-operator/step-registry/sdn/conf/multi-manifest/sdn-conf-multi-manifest-commands.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+cat >> "${SHARED_DIR}/manifest_cluster-network-03-config.yml" << EOF
```

Comment:
nit: `>>` -> `>` here?  Doesn't seem like we'd be appending to existing YAML, and if we do we'd need a `---` delimiter or some such, right?

Reply:
Hm... didn't notice, just copied it from the hybrid PR